images/build/Dockerfiles/Dockerfile

```diff
@@ -18,9 +18,12 @@ ENV RELEASE_TAG_NAME=${RELEASE_TAG_NAME}
 ENV DEBIAN_FLAVOR="stretch"
 
 RUN ./build.sh dotnetcore /opt/startupcmdgen/dotnet
+RUN ./build.sh golang     /opt/startupcmdgen/golang
+RUN ./build.sh hugo       /opt/startupcmdgen/hugo
```

Comment:
not needed?

Reply:
@arroyc may you elaborate on what you mean? Without this code, the binaries from `go build` that are needed in https://github.com/microsoft/Oryx/blob/21ef7ce9cd1b58b68a8e5da304c9ed0ff6bf1d7d/src/BuildScriptGenerator/DefaultRunScriptGenerator.cs#L92 will cause a failure

Reply:
im asking why do we need the hugo? do we have startupscriptgenerator for hugo?

Reply:
@arroyc currently there's no startupscriptgenerator for hugo, but I have added it in this PR

Reply:
unless we have startupscriptgenerator we should not add this

Reply:
we should only add what we have

Reply:
@arroyc work item 1394597 requests support for all minus Java. Are asking I remove all Hugo related code? https://github.com/microsoft/Oryx/pull/1149/commits/2cd6d2aa3f037ee05424eaaa711a5d4376e9cd22

Reply:
all im saying is if you add this line here that means we are assuming there is a startupscriptgenerator for hugo ... which is not there .. if you add it here I expect a startupscriptgenerator for hugo also present in this PR

Reply:
@arroyc hugo startupscriptgenerator is in this PR: https://github.com/microsoft/Oryx/blob/417e9d8bcf355dc203c9a7868ed00629d5f245cd/src/startupscriptgenerator/src/hugo/scriptgenerator.go

Are you asking the `orxy setupEnv` get added?